<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICPneumaticActionPage</class>
 <widget class="QWidget" name="ICPneumaticActionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="setButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="subArmGroupBox">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QCheckBox::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QCheckBox::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}



</string>
     </property>
     <property name="title">
      <string>Sub Arm</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="ICLineEditWithVirtualNumericKeypad" name="fbDelayLineEdit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="ICLineEditWithVirtualNumericKeypad" name="udDelayLineEdit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="forwardCheckBox">
        <property name="text">
         <string>Forward</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="backwardCheckBox">
        <property name="text">
         <string>Backward</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="upCheckBox">
        <property name="text">
         <string>Up</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="downCheckBox">
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="positionGroupBox">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}



</string>
     </property>
     <property name="title">
      <string>Position</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="verticalRadioButton">
        <property name="text">
         <string>Vertical</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="horizonRadioButton">
        <property name="text">
         <string>Horizon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Delay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ICLineEditWithVirtualNumericKeypad" name="posistionDelayLineEdit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
